src/test/unittest/unittest.sh

```diff
@@ -78,6 +78,7 @@ export UNITTEST_LOG_LEVEL GREP TEST FS BUILD CHECK_TYPE CHECK_POOL VERBOSE SUFFI
 
 VMMALLOC=libvmmalloc.so.1
 TOOLS=../tools
+LIB_TOOLS="../../tools"
```

Comment:
Not strictly related with Your changes, but I think to be consistent - "$LIB_TOOLS", could be also used here:
https://github.com/pmem/pmdk/blob/48b3dd09493e11fdef8aa2f39e687b5a494228c5/src/test/unittest/unittest.sh#L82-L83